runner/ansible/README.md

```diff
@@ -22,3 +28,102 @@ These are the fields needed by Trento:
 is that the labels are used for control purpose (select all the checks with this label e.g.), and the groups are used just for visual purposes
 - `description`: A longer description about the test purpose. It can be written using markdown.
 - `implementation`: Usually the task `main.yml` content
+
+## Check files
+
+The check files contain the actual task which the runner will execute. An
+example is available at [tasks/main.yml](roles/checks/1.1.1/tasks/main.yml).
+These files are written like a normal ansible task.
+
+## Check structure
+
+The checks folder is in `runner/ansible/roles/checks`. Each check has its own
+number (e.g. 1.1.1) which refers to its place in the queue (1.1.2 will be 
+executed after 1.1.1 etc.). Inside these numbered folders are two subfolders, both of these each has one file in them named `main.yml`:
+- `defaults`
+   The defaults directory (or better the main.yml file inside of it)
+   contains all the required [metadata](#metadata-files) for the check. 
+
+- `tasks`
+
+The checks themselfs are written like any ansible task, however their 
+output is either `true` (check passed) or `false` (check failed). This 
+output will then be passed to the `post-results` block with the `status` 
+field containing the result of the check. 
+Following lines can/should be left as they are since they are part of 
+how the checks are executed:
+
+```
+name: "{{ id }}.check"
+```
+
+```  
+check_mode: no
+register: config_updated
+changed_when: config_updated.stdout != expected[id]
+```
+```
+- block:
+    - import_role:
+        name: post-results
+  when:
+    - ansible_check_mode
+  vars:
+    status: "{{ config_updated is not changed }}"
+``` 
+
+
+After implementing your checks, trento needs to be rebuild, moved to the 
+target machine and before beeing started, all running services related to
+trento need to be stopped.
+
+## Examples
+
+As an example (we will use 3.0.0), lets add a test which simply creates a simple file in /tmp.
```

Comment:
I wouldn't reference any existing test (it might go away at some point). Make an example with the `dummy` name or similar
Besides this, I would actually implement a real check (which checks something).
Like check the file `/tmp/my_file` exists or similar.
We need to emphasize that the check must not apply any change in the machine, itself. It is a `read-only` operation